server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueue.java

```diff
@@ -119,4 +123,13 @@ public String toString() {
             .add("name", name)
             .toString();
     }
+
+    public void rebuild() {
+        boolean parallel = false;
+        Mono.fromRunnable(dequeuer::queuePurge)
+            .thenMany(
+                Flux.fromStream(StreamSupport.stream(Spliterators.spliteratorUnknownSize(browse(), Spliterator.ORDERED), parallel))
```

Comment:
Use directly the underlying daos to access mail queue references? (See previous comments)